ruby/README.md

```diff
@@ -51,8 +51,16 @@ Build the image:
 docker build -t username/imagename:latest .
 ```
 
-Push it to Docker Hub:
+Push it to DockerHub:
 
 ```sh
 docker push username/imagename
 ```
+### 4. Push to Iron.io 
+
+When running as a worker, push via [`iron` CLI](https://github.com/iron-meteor/iron-cli#usage): 
```

Comment:
Wrong URL, that's for a different project, should be: https://github.com/iron-io/ironcli